.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
Yes – the reason I changed it, is because I think it's great when you can just check-out a repo (`bash-it` in this case), and just run the tests, and have them succeed.

Until this change, running `test/run` would break because `bats` wasn't there.  I had to look at `travis.yml` to figure out what was going on.  

So it seemed that moving this logic from the `travis.yml` to the test runner itself made sense.  Now anyone can just run `test/run` and have it work.